MdePkg/Library/BasePanicLibNull/PanicLib.c

```diff
@@ -0,0 +1,40 @@
+/** @file
+  Base Panic Library with empty functions.
+
+  Copyright (c) 2023, Microsoft Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Base.h>
+#include <Library/PanicLib.h>
+
+//
+// Define the maximum panic message length that this library supports
+//
+#define MAX_PANIC_MESSAGE_LENGTH  0x100
+
+/**
+  Prints a panic message containing a filename, line number, and description.
+  This is always followed by a dead loop.

```

Comment:
This statement isn't true in the null case, probably worth calling out.

Reply:
Updated for the NULL implementation.